src/resource/notes/notes.service.ts

```diff
@@ -0,0 +1,58 @@
+import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { NotesEntity } from 'src/database/entities/notes.entity';
+import { Repository } from 'typeorm';
+import { NoteDTO, editNoteDTO } from './dto';
+import { UserEntity } from 'src/database/entities/user.entity';
+
+@Injectable()
+export class NotesService {
+    constructor(
+        @InjectRepository(NotesEntity)
+        private notesRepository: Repository<NotesEntity>,
+        @InjectRepository(UserEntity)
+        private userRepository: Repository<UserEntity>,
+    ) {}
+
+    async fetchAllNotes(userId: number) {
+        const user = await this.userRepository.findOneBy({ id: userId });
```

Comment:
We can get user from Guard decorator, because it is in request data and we don't have to search the user in the database.

Reply:
Hint: Look for endpoint: GET : users/me